docs/references/nextjs/auth.mdx

```diff
@@ -32,54 +65,72 @@ import { auth } from '@clerk/nextjs';
 
 export async function GET() {
   const {userId, getToken} = auth();
+
```

Comment:
I know this isn't part of your changes, but should this code block really be jsx/tsx, instead of js/ts? I thought it wasn't normal to add the x unless it was a component

Reply:
This is a Route Handler -- it definitely should be `route.[js/ts]`. Route Handlers don't return JSX/TSX. I also believe we have landed on presenting just TypeScript so it probably should be `route.ts`